src/v/cluster/health_monitor_types.h

```diff
@@ -67,14 +67,24 @@ struct partition_status
     size_t size_bytes;
     std::optional<uint8_t> under_replicated_replicas;
 
+    /*
+     * estimated amount of data above local retention that is subject to
+     * reclaim under disk pressure. this is useful for the partition balancer
+     * which is interested in free space on a node. a node may have very little
```

Comment:
I think it makes sense to use the local retention value because from the perspective of the partition balancer, space consideration can be reduced to what things would look like if we space management weren't enabled: partitions would mostly consume space at roughly their local retention setting.

Just to be sure, if we're thinking about _actually_ reclaimable space, would it make sense to consider the low-space-reclaimable data? It would feel weird to, since it reduces down to one or two segments per partition, but still seems like something to consider

Reply:
>actually reclaimable space

when is it not actually reclaimable?

Reply:
I mean that the local retention bytes is an underestimate for the total amount of reclaimable space if space management is on, right? We could reclaim all data up until the first two segments in low space scenarios (vs just reclaiming up to the local retention point)

Reply:
ahh i see. yeh, i think that we want to avoid advertising to the balancer that it could assume data is reclaimable past local retention. the reasoning is that local retention is our guideline for a healthy amount of data per partition. ignoring that the balancer might be making decisions that put some node into an unhealthy state. at least that is my reasoning. does that make sense?

Reply:
Yeah, makes sense. I agree, using local retention is fairly intuitive since it kind of reduces back to what we had pre-space-management.